extensions/amp-lightbox/1.0/lightbox.js

```diff
@@ -163,7 +170,16 @@ function LightboxWithRef(
         layout={true}
         paint={true}
         part="lightbox"
-        wrapperClassName={`${classes.defaultStyles} ${classes.wrapper}`}
+        contentStyle={
+          // Prefer style over class to override `ContainWrapper`'s overflow
```

Comment:
Hmm. Why do we need both `contentStyle ` and `wrapperClassName ` for propagation of these two styles?

Reply:
A couple notes here:  
(1) `ContainWrapper` applies `overflow: hidden` to the content element's `style` 
(2) We apply `containScroll` class in different layers depending on `scrollable`

When `scrollable`:
- `contentStyle` is needed to overwrite `overflow: hidden` given by `ContainWrapper` - `overscrollBehavior: none` can be added via `contentClass` instead. 
- The same could be expressed with `contentClass={scrollable ? classes.containScroll : ''} contentStyle={scrollable ? {overflow: scroll} : {}}` But I simplified the application of `overscroll-behavior` to one `contentStyle` since that prop was necessary to override the default described in (1).

When `!scrollable`:
- `wrapperClassName` is needed for when there may or may not be overflowing content but we do not allow the lightbox to scroll - thereby keeping the `overflow: hidden` on the content element created by `ContainWrapper` but setting `overflow: scroll; overscroll-behavior: none;` on the wrapper element to lock scrolling. 
- `overflow: hidden` causes the element to disregard `overscroll-behavior`. So, the above `overflow: scroll` on the wrapper element is a bit misleading since the wrapper element does not actually scroll, but is needed to allow `overscroll-behavior` to be effective.
- Since this comment I've changed the `overflow: scroll` on the wrapper element to `overflow: auto` to yield the same result with hopefully less confusion.

Reply:
An alternative could be to apply `classes.containScroll` to `wrapperClass` in all cases, but seeing as it's not necessary when `scrollable`, I thought that would raise the same question. 😅  I was hoping the syntax in the `contentStyle` and `wrapperClassName` props would help explain that only one or the other is applied depending on `scrollable`.

Reply:
Ok. Got it.